src/vcastTestInterface.ts

```diff
@@ -902,4 +902,4 @@ export async function openCodedTest(testNode: testNodeType) {
   if (fs.existsSync(testNode.testFile)) {
     openFileWithLineSelected(testNode.testFile, testNode.testStartLine - 1);
   }
-}
+}
```

Comment:
I think we can revert this?

Reply:
This a new line at the end of file, but yeah, we can revert if you want

Reply:
Just to make it cleaner on the diff.

Then we can be sure there's no "application" changes in this.